aiohttp/web_request.py

```diff
@@ -444,7 +448,12 @@ def remote(self) -> Optional[str]:
 
     @reify
     def url(self) -> URL:
-        url = URL.build(scheme=self.scheme, host=self.host)
+        """The full URL of the request."""
+        host, has_port, port = self.host.partition(":")
```

Comment:
This isn't sufficient because it could be an IPv6 and we need to handle brackets

Reply:
https://github.com/aio-libs/yarl/blob/38ef0f95d25c240e17963971a41ed85b0b0c4e31/yarl/_url.py#L1089